ui/DesignSystem/Component/WalletStatusIndicator.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="typescript">
+  import { createEventDispatcher } from "svelte";
+  import { store, Status } from "ui/src/wallet";
+  import { store as transactions } from "ui/src/transaction";
+
+  import { Icon } from "../Primitive";
+  import Tooltip from "./Tooltip.svelte";
+  import SidebarItem from "./SidebarItem.svelte";
+
+  export let active: boolean;
+
+  const dispatch = createEventDispatcher();
+  const onClick = () => {
+    dispatch("walletClick");
+  };
+
+  $: pendingTxs = $transactions.filter(
+    tx => tx.status === "Awaiting inclusion"
+  );
+  $: wallet = $store;
+  // Hack to have Svelte working with checking the $wallet variant
+  // and thus be able to access its appropriate fields.
+  $: w = $wallet;
```

Comment:
This hack is not necessary anymore. You can just use `$wallet` instead of `w`.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1930/commits/3f87e87b845bdb63021212e7cffd4fe17d2adbca